### DISCUSSION TITLE
Feedback for course "Java, IntelliJ IDEA and Maven"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#3] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by MolinaElGringo** [comment]
The tutorial was I bit difficult (mostly when it comes to download everything). I still manage to do it by asking my friends.

**Reply by ludelafo** [reply]
Thank you for your feedback!

You will have to set up a few IntelliJ IDEA + Maven projects for the rest of the course (for the practical works for example). Hopefully, this will help you to learn and integrate all the concepts little by little.

Feel free to re-do the practical content as much as you want and do not hesitate to contact us if you need any feedback on your work or if you have difficulties.

**Comment by Cobora2001** [comment]
I was able to complete it, though I underestimated the time it would take.

I'm not sure that I understood everything I did though. A lot of this process in the Intellij interface was following the instructions, copying some stuff in different places, and so on. At this time, I don't yet understand the content of the modifications I did, such as in the pom.xml for example.

If this is by design, and will be approached later, then it's not a problem, though it may be nice to have it mentionned (unless I missed it somewhere, I tried to go fast after all). If I'm supposed to have understood everything I did, I'll have to go through everything (at least the configuration parts) once again to gain a better understanding.

Well, onto the next thing !

Edit:
Also, regarding lign 870 and 946, I was unsure about what to do with this information, not knowing if I was supposed to do something with this information, like following the links and installing whatever was on the other side, or if it was just more information.

It turned out to be the second, but I think it could gain from being clearer, especially when another step of the process fails for us and that we have to come back and troubleshoot what might have gone wrong. We may fall on that and explore this path when this isn't what was causing the issue.

**Reply by ludelafo** [reply]
Thank you for your comment!

We are aware that this course has many new things to learn for many of you. Some elements will be seen in depth later during this course, some will be seen in future courses during your studies.

We did try to give an insight on how Maven works but it is true we do not have the time to go much further than this. If you read the course material, then check the practical content again, some things should be clearer for you. If not, always feel free to leave us a comment on the exact thing we could improve.

> Also, regarding lign 870 and 946, I was unsure about what to do with this information, not knowing if I was supposed to do something with this information, like following the links and installing whatever was on the other side, or if it was just more information.

Could you point us to the exact file you are talking about in this comment? If you open the Markdown file, you can then select the exact line(s) you want to highlight and copy/paste the URL from your browser to share them to us.

**Reply by Cobora2001** [reply]
Hi,

Yes sorry regarding the ligns, I meant in the instructions for this section:

- [Lign 870](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/40274e6ed15de6d7e04ec4e1c8cad8e95e85a05a/04-java-intellij-idea-and-maven/COURSE_MATERIAL.md?plain=1#L870)
- [Lign 946](https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/40274e6ed15de6d7e04ec4e1c8cad8e95e85a05a/04-java-intellij-idea-and-maven/COURSE_MATERIAL.md?plain=1#L946)

Thanks for the response,

Thomas

**Comment by ludelafo** [comment]
All previous comments have been marked as hidden (outdated) for the start of the new DAI semester.
